packages/astro/src/react/controlComponents.tsx

```diff
@@ -24,7 +24,7 @@ export function SignedIn(props: PropsWithChildren) {
   return props.children;
 }
 
-const $isLoadingClerkStore = computed($clerkStore, clerk => clerk?.loaded);
+const $isLoadingClerkStore = computed($csrState, state => state.isLoaded);
```

Comment:
@panteliselef I replaced this to use the `isLoaded` property from the CSR state, since it gives more accurate status and we set the value right after Clerk has been loaded [here](https://github.com/clerk/javascript/blob/944bb2e23ccd6187bf1ad623b48629e8c2b71442/packages/astro/src/internal/create-clerk-instance.ts#L35).

The `$clerkStore` gives unreliable loaded status, maybe because we're setting it only once [here](https://github.com/clerk/javascript/blob/944bb2e23ccd6187bf1ad623b48629e8c2b71442/packages/astro/src/internal/create-clerk-instance.ts#L27)? And for some reason, nanostore freezes the object?

Even though the tests are passing on my local, it is still failing on CI 🤔